config/packages/api_platform.yaml

```diff
@@ -1,15 +1,43 @@
 api_platform:
-    # The title of the API.
+    # Documentation
     title: 'DDB Cover Service'
+    description: >-
+        This service provides covers for library materials indexed by isbn, issn, faust, pid.
+        The service is provided by [Danskernes Digitale Bibliotek](https://www.danskernesdigitalebibliotek.dk/)
 
-    # The description of the API.
-    description: 'Search for covers by ISBN/FAUST/PID'
+        ### Authentication notes
 
-    # The version of the API.
+        Authentication is done via OAuth2 against auth.dbc.dk. To obtain a valid token follow instructions in
+        [1.2. Password Grant](https://github.com/DBCDK/hejmdal/blob/master/docs/oauth2.md#12-password-grant).
+        To use the "Authorize" option in this tool use your 'client_id' and 'client_secret' and fill in '@' for
+        both username and password.
+
+        ### Implementation notes
+
+        Currently the API is not fully implemented. The following features are missing:
+
+        * Generic Covers: parameter is currently ignored. Service doesn't yet provide generic covers.
```

Comment:
We do have generic cover(s). Aka default cover in the soap endpoint. When SOAP is removed and client move to the rest endpoint they may need a way to ask for this. I left the text here because we need to address this question.